find.go

```diff
@@ -0,0 +1,61 @@
+package godash
+
+import (
+	"fmt"
+	"reflect"
+)
+
+// Find out elements.
+//
+// Input of type slice is supported as of now.
+// Output is a elements are matched.
+// PredicateFn function is applied on each element of input to determine to find element
```

Comment:
// PredicateFn function is applied on each element of input to determine to find element until it finds the element